### PR TITLE
#19 & #21

### PR DESCRIPTION
**Pull request by petelapczynski**
stories for Beer Card updates used on Home and Browse routes. Added SuggestedGlass to Beer data model.

### CODE REVIEW

File: Webrew.Data/Collections/ReviewCollection.cs [hunk 1]
```diff
@@ -1,18 +1,26 @@
...
+    {
+        private IMongoQueryable<Review> Reviews { get { return Collection.AsQueryable(); } }
+
+        public ReviewCollection(IDbClient client) : base(client)
+        {
+        }
+
+        public Task<Review> UpdateReview(ObjectId id, Review review)
+        {
+            throw new NotImplementedException();
```
Comment (paul-lapczynski) [inline Webrew.Data/Collections/ReviewCollection.cs]: Whoops, did you not hook this to the collection?

File: Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html [hunk 2]
```diff
@@ -1,13 +1,25 @@
...
+    <mat-card-header >
+      <mat-card-title>{{ beer.name }}</mat-card-title>
+      <mat-card-subtitle>{{ beer.company }}</mat-card-subtitle>
+    </mat-card-header>
+    <a href="/beer/{{beer.id}}">
+      <img class="card-image" src="{{ beer.imageURL }}" alt="Photo of {{ beer.name }}" >
+    </a>
+  <mat-card-content>
+    <hr />
+    <div mat-card-style class="float-left">Style: {{ beer.style }}</div>
```
Comment (paul-lapczynski) [inline Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html]: Avoid floating. Use flex or fxFlex directives

File: Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html [hunk 3]
```diff
@@ -1,13 +1,25 @@
...
+    <div mat-card-abv class="float-center">ABV: {{ beer.abv }}%</div>
+    <hr />
+    <div>
+      <p mat-card-desc>{{ beer.description }}</p>
+      <p mat-card-desc>Suggested Glass: {{ beer.suggestedGlass }}</p>
+    </div>
+  </mat-card-content>
+  <mat-card-footer class="footer">
+    <p>
+      Added {{ beer.createdDate  | date: "longDate"}} by <a href="/userprofile/{{ beer.createdByUser }}" class="">{{ beer.createdByUser }}</a>
```
Comment (paul-lapczynski) [inline Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html]: With links in angular, use the [routerLink] attribute so it doesn't redirect

File: Webrew.Client/ClientApp/src/app/browse/components/browse/browse.component.ts [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+@Component({
+    selector: 'browse',
+    templateUrl: './browse.component.html',
+    styleUrls: ['./browse.component.scss']
+})
+export class BrowseComponent implements OnInit {
+    constructor(private beersService: BeersService) {}
+
+    beers$;
```
Comment (paul-lapczynski) [inline Webrew.Client/ClientApp/src/app/browse/components/browse/browse.component.ts]: Give this a type

File: Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html [hunk 5]
```diff
@@ -1,13 +1,25 @@
...
+      <img class="card-image" src="{{ beer.imageURL }}" alt="Photo of {{ beer.name }}" >
+    </a>
+  <mat-card-content>
+    <hr />
+    <div mat-card-style class="float-left">Style: {{ beer.style }}</div>
+    <div mat-card-avgrating class="float-right">Avg Rating: {{ beer.avgRating }}</div>
+    <div mat-card-abv class="float-center">ABV: {{ beer.abv }}%</div>
+    <hr />
+    <div>
+      <p mat-card-desc>{{ beer.description }}</p>
```
Comment (paul-lapczynski) [inline Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html]: These attributes like "mat-card-desc" and "mat-card-abv" I'm pretty sure don't do anything unless it's an angular 8 thing

File: Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html (hunk 3 above)
Comment (petelapczynski) [reply Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html]: I tried routerLink and couldn't figure it out

File: Webrew.Client/ClientApp/src/app/browse/components/browse/browse.component.ts (hunk 4 above)
Comment (petelapczynski) [reply Webrew.Client/ClientApp/src/app/browse/components/browse/browse.component.ts]: this is just fake till it gets fully implemented, it needs to handling filtering but I wasn't sure how to do that.

File: Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html (hunk 5 above)
Comment (petelapczynski) [reply Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html]: I wasn't sure how you had named the previous ones so I had just named each after the data that was in it.

File: Webrew.Data/Collections/ReviewCollection.cs (hunk 1 above)
Comment (petelapczynski) [reply Webrew.Data/Collections/ReviewCollection.cs]: not quite sure, VS had yelled at me until I added that.

File: Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html (hunk 2 above)
Comment (petelapczynski) [reply Webrew.Client/ClientApp/src/app/display-card/components/display-card/display-card.component.html]: where do those go?

**Pull request merged**